nameko/standalone/events.py

```diff
@@ -31,6 +34,11 @@ def event_dispatcher(nameko_config, **kwargs):
     def dispatch(service_name, event_type, event_data):
         conn = Connection(nameko_config[AMQP_URI_CONFIG_KEY])
 
+        if KOMBU_SERIALIZER_CONFIG_KEY in nameko_config.keys():
+            serializer = nameko_config[KOMBU_SERIALIZER_CONFIG_KEY]
+        else:
+            serializer = DEFAULT_KOMBU_SERIALIZER
```

Comment:
Yes, they are all different with regard to None.
I'm not sure yet whether to allow None at all,  or where to allow it. I'll
have a new variant, which seems to be a bit cleaner, I will post it after
fixing the tests
On 27 Apr 2015 20:17, "David Szotten" <notifications@github.com> wrote:

> In nameko/standalone/events.py
> <https://github.com/onefinestay/nameko/pull/244#discussion_r29168017>:
>
> > @@ -31,6 +34,11 @@ def event_dispatcher(nameko_config, **kwargs):
> >      def dispatch(service_name, event_type, event_data):
> >          conn = Connection(nameko_config[AMQP_URI_CONFIG_KEY])
> >
> > +        if KOMBU_SERIALIZER_CONFIG_KEY in nameko_config.keys():
> > +            serializer = nameko_config[KOMBU_SERIALIZER_CONFIG_KEY]
> > +        else:
> > +            serializer = DEFAULT_KOMBU_SERIALIZER
>
> foo.get(key) or default is different to foo.get(key, default) if e.g. foo[key]
> == None # or 0
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/onefinestay/nameko/pull/244/files#r29168017>.
>